colossalai/shardformer/policies/bert.py

```diff
@@ -176,6 +176,13 @@ class BertModelPolicy(BertPolicy):
     def __init__(self) -> None:
         super().__init__()
 
+    def module_policy(self):
+        module_policy = super().module_policy()
+        from transformers.models.bert.modeling_bert import BertModel
+        module_policy[BertModel] = ModulePolicyDescription(
+            method_replacement={'forward': partial(bert_model_forward, stage_manager=self.pipeline_stage_manager)})
```

Comment:
Should check if pipeline parallelism is enabled?

Reply:
I am sorry that I did not push the updated code in time, please see the new commit!

Reply:
Good!